verisigndns/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'java'
+apply plugin: 'checkstyle'
+
+sourceCompatibility = 1.6
+
+test {
+  systemProperty 'verisigndns.url', System.getProperty('verisignmdns.url', '')
+  systemProperty 'verisigndns.username', System.getProperty('verisignmdns.username', '')
+  systemProperty 'verisigndns.password', System.getProperty('verisignmdns.password', '')
+  systemProperty 'verisigndns.zone', System.getProperty('verisignmdns.zone', '')
+}
+
+dependencies {
+  compile      project(':denominator-core')
+  compile     'com.netflix.feign:feign-core:8.10.0'
+  compile     'com.netflix.feign:feign-sax:8.10.0'
+  compile     'com.google.guava:guava:18.0'
```

Comment:
guava is not a safe dependency to add to any denominator first-party
provider. the version skew in deployments is too wide.